server/src/apub/fetcher.rs

```diff
@@ -376,7 +376,10 @@ pub async fn get_or_fetch_and_insert_post(
   }
 }
 
-fn upsert_comment(comment_form: &CommentForm, conn: &PgConnection) -> Result<Comment, LemmyError> {
+pub fn upsert_comment(
```

Comment:
Not really related to this PR, but these upsert functions should be moved to lemmy_db.

Reply:
I was thinking that too, I'll actually do that here.